react/features/notifications/components/web/NotificationsContainer.js

```diff
@@ -153,7 +153,9 @@ class NotificationsContainer extends Component<Props> {
      * @inheritdoc
      */
     componentDidMount() {
-        this._updateTimeouts();
+        if (navigator.product === 'ReactNative') {
```

Comment:
This container is only used on web, so you can remove the check and delete all associated functions.